src/AppBundle/Entity/Post.php

```diff
@@ -80,10 +82,21 @@ class Post
      */
     private $comments;
 
+    /**
+     * @var Collection|Tag[]
```

Comment:
Sure, but you then tie code using the entities to your internal implementation and I don't think we should encourage that.